src/main/java/org/icgc_argo/workflow/search/controller/RunsApi.java

```diff
@@ -243,4 +243,33 @@ Mono<ResponseEntity<RunListResponse>> listRuns(
             response = ErrorResponse.class)
       })
   Mono<ResponseEntity<RunId>> cancelRun(@Valid @RequestBody String runId);
+
+  @ApiOperation(
+      value = "Trigger the scheduler to fetch and initialize runs",
+      nickname = "Manual trigger schedule",
+      notes =
+          "This endpoint causes the management scheduler to fetch runs from the management database and set their state to INITIALIZING.\n\n"
+              + "After the state change, the runs are published on the gatekeeper-in-queue.\n\n"
+              + "This method publishes a Trigger String message onto the Trigger-queue then read by the scheduler, initiating the fetch\n\n",
+      response = String.class,
+      tags = {
+        "WorkflowExecutionService",
```

Comment:
can we change this to a new tag called `WES Management`?

The WES is a specific interface that we adhere to, so extra changes like this should be listed separately: https://ga4gh.github.io/workflow-execution-service-schemas/docs/#tag/WorkflowExecutionService